concordium-node/src/configuration.rs

```diff
@@ -511,6 +509,14 @@ pub struct ConnectionConfig {
         env = "CONCORDIUM_NODE_CONNECTION_DEDUPLICATION_HASHING_ALGORITHM"
     )]
     pub deduplication_hashing_algorithm: DeduplicationHashAlgorithm,
+    #[structopt(
+        long = "max-normal-keep-alive",
+        help = "Max seconds to hold connection to dead \"normal\" node before discarding. Must be \
+                at least 3 times greater than the value of \"housekeeping-interval\".",
+        default_value = "120",
+        env = "CONCORDIUM_NODE_MAX_NORMAL_KEEP_ALIVE"
+    )]
+    pub max_normal_keep_alive: u64,
```

Comment:
Minor, but u64 is probably overkill.

Reply:
True.. u16 will hold ~17 hours in seconds, which will probably suffice?

Reply:
I would stick with u64 just for consistency with other time parameters.

Reply:
I'm fine with this.

Reply:
I'll stick to u64 to avoid needing to cast and increase readability of the code.